app/controllers/events_controller.rb

```diff
@@ -30,6 +30,18 @@ def create
     end
   end
 
+  def update
+    @event = Event.find(params[:id])
+    authorize @event
+
+    if @event.update(event_params)
+      flash[:notice] = 'Event was successfully updated.'
+      redirect_to @event
+    else
+      render 'edit'
```

Comment:
Also I realized that the flash notice on success wasn't actually being rendered anywhere so I added that into events#show view.